python/cuml/cuml/svm/svc.pyx

```diff
@@ -511,32 +558,14 @@ class SVC(SVMBase,
 
     def _fit_proba(self, X, y, sample_weight) -> "SVC":
         from sklearn.calibration import CalibratedClassifierCV
-        from sklearn.multiclass import OneVsOneClassifier, OneVsRestClassifier
-
-        params = self.get_params()
-        params["probability"] = False
-
-        # Ensure it always outputs numpy
-        params["output_type"] = "numpy"
 
         # Currently CalibratedClassifierCV expects data on the host, see
         # https://github.com/rapidsai/cuml/issues/2608
         X = input_to_host_array_with_sparse_support(X)
         y = input_to_host_array(y).array
 
-        if self.n_classes_ == 2:
-            estimator = SVC(**params)
-        else:
-            if self.decision_function_shape == 'ovr':
-                estimator = OneVsRestClassifier(SVC(**params))
-            elif self.decision_function_shape == 'ovo':
-                estimator = OneVsOneClassifier(SVC(**params))
-            else:
-                raise ValueError
-
-        self.prob_svc = CalibratedClassifierCV(estimator,
-                                               cv=5,
-                                               method='sigmoid')
+        params = {**self.get_params(), "probability": False, "output_type": "numpy"}
+        self.prob_svc = CalibratedClassifierCV(SVC(**params), ensemble=False)
```

Comment:
Thanks for spotting this! It looks like it was a bug. The `fit` function (with `probability=False`) should call the `_fit_multiclass` function that should in turn call the `MulticlassClassifier` and `OneVsRestClassifier`/`OneVsOneClassifier`. No need for OVR/OVO here. And using `CalibratedClassifierCV` with `ensemble=False` should ensure that we have a single calibrated estimator to transfer.